docs/cadence_migration_guide/staging-guide.mdx

```diff
@@ -44,3 +44,13 @@ flow-c1 migrate is-staged HelloWorld --network=testnet
 ```
 
 A response of true indicates that your contract has been approved by the Flow Blockchain Testnet network and is ready for the migration process.
+
+### Validate your contract
+
+To validate your contract, execute the following command:
```

Comment:
The sentence below explains what is actually returned, but the phrasing here makes it seem like the command performs the validation when the command is run. Maybe rephrase:

 ```suggestion
### Check the validation status
 
To check the validation status of your contract update, execute the following command:
```